docs/meshcloud.project-metering.md

```diff
@@ -75,13 +75,22 @@ This export will contain the line items (see above) of all the chargeback statem
 The line item data is suitable for feeding into chargeback processing, e.g. importing it to an ERP System to transfer
 budgets between cost centers.
 
-Chargeback Statements also contain billing information per line item. Your Cloud Foundation team can [configure](meshstack.billing-configuration.md)
+Chargeback Statements also contain billing information per line item. Your platform team can 
+[configure](meshstack.billing-configuration.md)
 which information meshStack should include as billing information in chargeback statements.
 
-> Cloud Foundation teams typically configure billing information to payment method name, identifier, expiration date and amount as well as any workspace tags, project tags and payment method tags.
+> Platform teams typically configure billing information to payment method name, identifier, 
+> expiration date and amount as well as any workspace tags, project tags and payment method tags.
 
 You can review this billing information in meshPanel when opening chargeback statement. CSV Exports of chargeback statements also include the configured billing information.
 
+For customers that export chargeback statements on a monthly basis, the following recommendation applies:
+Chargeback statements of the previous month should be processed not earlier than the `finalizeReportsAfterDays` setting,
+plus 2 days, at 7 am (German local time). For example, if `finalizeReportsAfterDays` is set to 4 
+(which is the default), then chargeback statements of May can be processed starting at June 6th, 7am.
```

Comment:
regarding the 7 am: the cron job actually runs at 04:33, but: this is the system time zone, which is probably UTC, so this will translate to 05:33 or 06:33 (depending on whether we have daylight savings time or not). We don't want to annoy our customers with such technical details, so we just say 07:00, which should be safe regardless of whether we have daylight savings time or not.